data/calculated.py

```diff
@@ -24,7 +21,7 @@ def __init__(self, url: str, **kwargs):
         Keyword Parameters:
         calculated -- a dict of the calculated variables.
         """
-        super(CalculatedData, self).__init__(url, **kwargs)
+        super().__init__(url, **kwargs)
```

Comment:
This `2to3` script really sucks at it's job lol...
The `super()`-with-arguments is a Python 2 relic for anyone interested: https://stackoverflow.com/a/17509938/2231969